### PR TITLE
ISPN-5473 Test suite logging improvements

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-5473

### CODE REVIEW

File: core/src/main/java/org/infinispan/statetransfer/StateResponseCommand.java [hunk 1]
```diff
@@ -102,6 +102,7 @@ public void setParameters(int commandId, Object[] parameters) {
    public String toString() {
       return "StateResponseCommand{" +
             "cache=" + cacheName +
+            "stateChunks=" + stateChunks +
```
Comment (anistor) [inline core/src/main/java/org/infinispan/statetransfer/StateResponseCommand.java]: missing ```,```

File: core/src/test/java/org/infinispan/context/InvocationContextTest.java [hunk 2]
```diff
@@ -96,9 +96,8 @@ public void run() {
       th.join();
       tm.resume(tx);
       tm.rollback();
-      assert throwables.size() == 1;
 
-      for (Throwable thr : throwables) thr.printStackTrace();
+      assert throwables.size() == 1;
```
Comment (galderz) [inline core/src/test/java/org/infinispan/context/InvocationContextTest.java]: Can we use `assertEquals` instead? Gives information about received value if it fails.

Comment (danberindei) [inline core/src/test/java/org/infinispan/context/InvocationContextTest.java]: Sure, I'll change the lines below as well

Comment (danberindei) [inline core/src/test/java/org/infinispan/context/InvocationContextTest.java]: Updated

**Comment by galderz** [comment]
Integrated

**Pull request closed without merging**